tests/integration/operation_tests.cpp

```diff
@@ -68,6 +68,23 @@ class OperationTest :
     umpire::Allocator* dest_allocator;
 };
 
+class ZeroCopyTest :
+  public OperationTest
+{
+};
+
+TEST_P(ZeroCopyTest, Zero) {
```

Comment:
Why does this need to be `TEST_P`? Can't we just use `TEST`?

Reply:
Sorry @jdahm, I missed this comment when I closed this particular PR.  I have no answer for this.  I merely copy/pasted from what was previously there.  Perhaps @davidbeckingsale knows?